src/balatro_gym/cards/joker.py

```diff
@@ -83,12 +78,11 @@ def rarity(self) -> Rarity:
         return Rarity.COMMON
 
     def get_mult(self, scored_cards: Sequence[PlayingCard], state: BlindState, scored_hand: PokerHandType) -> int:
-        return sum([3 if card.suit == Suit.SPADES else 0 for card in state.hand])
+        return sum([3 if Suit.SPADES in card.suit else 0 for card in scored_cards])
 
 
 class JollyJoker(JokerBase):
-
-    _base_cost: int = 5
+    _base_cost: int = 3
```

Comment:
The `HasCost` protocol is named `_cost`, so we should probably rename either the protocol and downstream usages or the jokers